make/hotspot/lib/CompileJvm.gmk

```diff
@@ -88,6 +88,12 @@ DISABLED_WARNINGS_gcc := array-bounds comment delete-non-virtual-dtor \
     maybe-uninitialized missing-field-initializers parentheses \
     shift-negative-value unknown-pragmas
 
+ifeq ($(DEBUG_LEVEL), fastdebug)
+  ifeq ($(call And, $(call isTargetOs, linux) $(call isTargetCpu, aarch64)), true)
+    DISABLED_WARNINGS_gcc += stringop-overflow
```

Comment:
Do you think it would be better if we add some comments here?
```suggestion
    # False positive warnings for atomic_linux_aarch64.hpp on GCC >= 13
    DISABLED_WARNINGS_gcc += stringop-overflow
```

Reply:
I was also surprised to see the `dangling-pointer` warning just disappear, but I guess I'll take the win.

Thank you for the code suggestion. We have not been consistent when it comes to (not) adding comments to the disabled warnings but perhaps this is a good time to start.